src/v2/view-builder/internals/BaseForm.ts

```diff
@@ -84,7 +84,9 @@ export default Form.extend({
      * Rerender would clear infoContainer or views classes can clear it explicitly.
      */
     const infoContainer= '<div class=\'o-form-info-container\'></div>';
-    this.$el.find('.o-form-error-container').before(infoContainer);
+    this.$el.find('.o-form-error-container')
```

Comment:
Getting "Property 'omit' does not exist on type 'OktaUnderscore'.ts(2339)" on L134. Seems unrelated to this change?